client/src/pages/demo/components/index.tsx

```diff
@@ -32,6 +33,23 @@ const DemoComponent: React.FunctionComponent = () => {
         <br />
         <hr className="w-3/4 mx-auto" />
       </div>
+
+      <div className="p-4 border-b border-black">
+        <Dropdown>
+          <a href="#dropdown">Dropdown</a>
+        </Dropdown>
```

Comment:
- i don't understand why you put anchor tag inside dropdown component.
- you supposed to demonstrate the dropdown component and its props :bow: